src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -39,6 +41,13 @@ protected override void DisconnectHandler(UIButton platformView)
 			base.DisconnectHandler(platformView);
 		}
 
+		public override void PlatformArrange(Rect rect)
+		{
+			base.PlatformArrange(rect);
+
+			UpdateValue(nameof(IImage.Source));
```

Comment:
This feels very expensive for iOS. On a re-layout, we reload the image and then resize it. If the layout changes slightly, this will keep happening. I see in Xamarin.Forms we just use the insets and forced it to obey: https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.iOS/Renderers/ButtonLayoutManager.cs#L284

In reality, developers should be using the correct image size for their presentation. Loading a huge image just to render at 20x20 is a total waste of resources. But, in the absense of that, we should probably not be doing so much extra work for this. 

Is it possible to use the insets pattern for MAUI?

Reply:
I think I may like this less that the options below where we lie to the OS and "resize" the image.